packages/transition-backend/src/services/transitObjects/transitServices/ServiceDuplicator.ts

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright 2024, Polytechnique Montreal and contributors
+ *
+ * This file is licensed under the MIT License.
+ * License text available at https://opensource.org/licenses/MIT
+ */
```

Comment:
We probably need a header to explain what does into that file. 
We have many ServiceSomething.ts which could make it confusing to decide where to put stuff.